includes/functions.php

```diff
@@ -1633,3 +1712,91 @@ function is_actor( $data ) {
 
 	return false;
 }
+
+/**
+ * Get an ActivityPub embed HTML for a URL.
+ *
+ * @param string  $url        The URL to get the embed for.
+ * @param boolean $inline_css Whether to inline CSS. Default true.
+ *
+ * @return string|false The embed HTML or false if not found.
+ */
+function get_embed_html( $url, $inline_css = true ) {
```

Comment:
With this function it looks like we can remove `embed_get()`?
I liked the attempt in `embed_get()` to get an oembed response first, should we do that here, too?